src/OrchardCore/OrchardCore.Setup.Core/SetupService.cs

```diff
@@ -254,6 +256,8 @@ void reportError(string key, string message)
                     context.Errors[key] = message;
                 }
 
+                await HttpBackgroundJob.ExecuteAfterEndOfRequestAsync("JobTest", scope => Task.CompletedTask);
```

Comment:
Add a comment why this is necessary.

Reply:
Oops this is a remaining testing code triggering a job named "JobTest", I removed this line.